examples/common/maths.ads

```diff
@@ -1,11 +1,32 @@
-with GL.Types;
+
+with GL.Types; use GL.Types;
 
 package Maths is
+
+    subtype tDegree is Single;
+    subtype tRadian is Single;
```

Comment:
Subtypes are implicitly convertible into each other. To actually improve type safety, these both should be `new Single` types. Also, I don't like the `t` prefix. I have seen it used in Pascal and Visual Basic, but never in Ada. Simply `Degree` and `Radian` suffices.

If you named the types like that because of the conversion functions, they can be named either `Degrees` / `Radians` (plural; ) or perhaps `As_Degree` / `As_Radian`.